### PR TITLE
Disallow rotation of garage door

### PR DESCRIPTION
**Pull request by Niklp09**
Doesn't happen anymore
image

### CODE REVIEW

File: my_future_doors/framed.lua [hunk 1]
```diff
@@ -36,7 +36,7 @@ local function add_door(col)
 				{-0.625,     1.4375,-0.5625,  0.625,  1.625,  0.5625}, --top
 			}
 		},
-
+		on_rotate = screwdriver.disallow,
```
Comment (SmallJoker) [inline my_future_doors/framed.lua:39]: You might happen to be lucky to not trigger an error for relying on the mod loading order. Please add the `screwdriver` mod to the mod.conf file of each changed mod to ensure that the mod is present and loaded *before* these doors.

**Comment by SmallJoker** [comment]
1. `my_sliding_doors:door2a2` does not rotate properly.
2. `my_future_doors:door2a` cannot be rotated but would probably look good if enabled.

**Comment by Niklp09** [comment]
1. fixed
2. I don't think so
image

**Comment by SmallJoker** [comment]
Strange. `my_sliding_doors:jpanel_corner_*` do not rotate properly either. I suppose the reason for that is that they're not using the MTG `doors` API (if available).

**Comment by SmallJoker** [comment]
@Niklp09 `my_sliding_doors:jpanel_corner_*` do not rotate properly either.

**Comment by SmallJoker** [comment]
@Niklp09 `my_sliding_doors:jpanel_corner_*` do not rotate properly either. Either restrict the screwdriver actions too, or use `on_rotate` to set the node above/below to the same param2.

**Comment by Niklp09** [comment]
@SmallJoker Thanks, fixed.

**Review by SmallJoker — APPROVED** [review]
Works. Will merge in a few days unless there are objections.

**Pull request merged**